app/peerinfo/metrics.go

```diff
@@ -53,4 +53,11 @@ var (
 		Name:      "version_support",
 		Help:      "Set to 1 if the peer's version is supported by (compatible with) the current version, else 0 if unsupported.",
 	}, []string{"peer"})
+
+	peerMevEnabledGauge = promauto.NewGaugeVec(prometheus.GaugeOpts{
+		Namespace: "app",
+		Subsystem: "peerinfo",
+		Name:      "mev_enabled",
+		Help:      "Set to 1 if mev is enabled on this peer, else 0 if disabled.",
```

Comment:
Although this is about MEV, can we call this `builder_api_enabled` to be in line with the env var/flag naming? 

```suggestion
		Name:      "builder_api_enabled",
		Help:      "Set to 1 if builder-api is enabled on this peer, else 0 if disabled.",
```

Won't comment through the rest of this PR for simplicities sake.

Reply:
Accepted. Thank you!